spec/support/rspec_rails_4.1.14.1.rb

```diff
@@ -0,0 +1,7 @@
+# Fix for rspec-rails and rails 4.1.14.1
+# This can be removed as soon as doing so doesn't cause test errors.
+# See https://github.com/rspec/rspec-rails/issues/1532
+
+RSpec::Rails::ViewRendering::EmptyTemplatePathSetDecorator.class_eval do
```

Comment:
@benlovell Added a check for that. Hopefully the fix will be in the next few days anyway. But also this app is running a fairly old version of rspec-rails.